File: js/sumPairs.js
exports.sumPairs = function(array, target) {
    let result = [];	
    for (i = 0; i < array.length; i++){	
      for (let j = i + 1; j < array.length; j++){
        if (array[i] + array[j] === target){	
          result.push([array[i], array[j]]);	
        }	
      }	
    }	
    if (result.length > 0){	
        return result;	
    }	
    return('unable to find pairs');
};
//console.log(exports.sumPairs([1,2,3,4,5], 7))

File: python/sum_pairs.py
def sum_pairs(array, target):
    result = []
    for i in range(0, len(array)):
        for j in range(i + 1, len(array)):
            if (array[i] + array[j] == target):
                result.append([array[i], array[j]])
    if len(result) > 0:
        return result
    else:
        return ('unable to find pairs')

#print(sum_pairs([1,2,3,4,5], 9))